### PR TITLE
Add initial operator-sdk generated scaffolding

### PR DESCRIPTION
**Pull request by redhatrises**
- Update domain information

### CODE REVIEW

File: config/certmanager/certificate.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  namespace: system
+spec:
+  # $(SERVICE_NAME) and $(SERVICE_NAMESPACE) will be substituted by kustomize
+  dnsNames:
+  - $(SERVICE_NAME).$(SERVICE_NAMESPACE).svc
+  - $(SERVICE_NAME).$(SERVICE_NAMESPACE).svc.cluster.local
+  issuerRef:
+    kind: Issuer
+    name: selfsigned-issuer
+  secretName: webhook-server-cert # this secret will not be prefixed, since it's not managed by kustomize
```
Comment (shawndwells) [inline config/certmanager/certificate.yaml:25]: Is the certificate stuff needed for Falcon?

File: config/default/kustomization.yaml [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+# [CERTMANAGER] To enable cert-manager, uncomment all sections with 'CERTMANAGER'. 'WEBHOOK' components are required.
+#- ../certmanager
+# [PROMETHEUS] To enable prometheus monitor, uncomment all sections with 'PROMETHEUS'.
+#- ../prometheus
+
+patchesStrategicMerge:
+# Protect the /metrics endpoint by putting it behind auth.
+# If you want your controller-manager to expose the /metrics
+# endpoint w/o any authn/z, please comment the following line.
+- manager_auth_proxy_patch.yaml
```
Comment (shawndwells) [inline config/default/kustomization.yaml:31]: What's the benefit of /metrics being exposed without authentication?

**Review by shawndwells — COMMENTED** [review]
Looks to be a few things that we don't need (proxy, certs). Is this intended to create initial scaffolding (and these will be removed in future increments?) or will we need these services?

**Comment by redhatrises** [comment]
Merging per internal discussion with @isimluk

**Pull request merged**